packages/story-editor/src/utils/getCropParams.ts

```diff
@@ -13,12 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * External dependencies
+ */
+import type { MediaElement } from '@googleforcreators/elements';
+
 /**
  * Internal dependencies
  */
+import type { CropParams } from '../app/media';
```

Comment:
```suggestion
import type { CropParams } from '../app/media/types';
```

Otherwise when importing from `../app/media/` TS will complain about all of these files there not being typed.

Reply:
Originally had it like this. The linter changed it.